tests/acceptance/databases/deprovision-test.js

```diff
@@ -74,16 +75,26 @@ test('/databases/:id/deprovision will deprovision with confirmation', function(a
   stubStacks();
   stubOrganization();
 
-  signInAndVisit('/databases/'+databaseId+'/deprovision');
-  fillIn('input[type=text]', databaseName);
+  signInAndVisit(`/databases/${databaseId}/deprovision`);
+
+  andThen(function(){
+    fillIn('input[type=text]', databaseName);
+  });
   andThen(function(){
     $('input[type=text]').trigger('input');
   });
   click('button:contains(Deprovision)');
-  andThen(function(){
-    assert.ok(didDeprovision, 'deprovisioned');
-    assert.equal(currentPath(), 'dashboard.stack.databases.index');
+  andThen(() => {
+    assert.ok(didDeprovision, 'deprovisioned', 'received successful response');
+    assert.equal(currentPath(), 'dashboard.stack.databases.new',
+      'with no dbs, user is routed to create one');
   });
+  // TODO: Figure out why the success message isn't showing up
+  // andThen(function(){
+  //   alert = findWithAssert('.alert-success', 'success message shown');
+  //   assert.ok(alert.text().indexOf(successMessage) > -1,
+  //     `expect "${successMessage}" as success message`);
+  // });
```

Comment:
I've got to figure this out before merging, but I need to put it on the shelf for a bit. The success message doesn't render and I'm not sure why. Now that I'm deleting the database, the app routes to `.index`, since there are no databases, routing logic to forwards again to `.new`. This works fine in dev, but looks like its asking the user to log in at some point during the test.
Any ideas @sandersonet?